### PR TITLE
Adding wg repos

### PR DESCRIPTION
**Pull request by espenove**
Added the non-document files from the WG repos.
Will need cleanup by the different working groups, especially app-directory

### CODE REVIEW

**Comment by rikoe** [comment]
I think the structure we should follow is:
```
root-directory
├── docs
├── examples
├── src   
    ├── api
    ├── app-directory
    ├── context
    └── intents
└── website
```
This is a pretty standard structure for projects.

File: app-directory/.gitignore [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (rikoe) [inline app-directory/.gitignore]: We can keep this file for now, but ideally we should have one .gitignore at the top level, not maintain multiple ones in subfolders.

File: app-directory/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#
```
Comment (rikoe) [inline app-directory/.travis.yml]: I think we should not add this file for now, it is related to the build process for the appd standalone repo.

File: app-directory/LICENSE [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (rikoe) [inline app-directory/LICENSE]: This should DEFINITELY not be here. There is a license for FDC3, and it is not copyrighted by IHS Markit.

File: app-directory/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Application Directory API -- THIS IS NOT A RATIFIED FDC3 API or SPECIFICATION
```
Comment (rikoe) [inline app-directory/README.md]: This should also go, this has been replaced by the equivalent file in the doc folder.

If there are README files in the src folder, they should be about how to consume and contribute to the source code.

File: app-directory/specification/appd.yaml [hunk 5]
```diff
@@ -0,0 +1,295 @@
+openapi: 3.0.0
```
Comment (rikoe) [inline app-directory/specification/appd.yaml]: I am tempted to move this file into a `appd-spec.yaml` file at the root of the appd source, to make it nice and explicit. But leave it as is for now, we need Frank's input.

File: context/src/examples/Contact.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+interface Contact extends Context {
```
Comment (rikoe) [inline context/src/examples/Contact.ts]: I am not sure where these examples should go, into a general examples folder from the root, or inside the src for context data. Leave them as a subfolder for now I guess.

File: app-directory/.gitignore (hunk 1 above)
Comment (espenove) [reply app-directory/.gitignore]: jepp - quick copy error

File: app-directory/README.md (hunk 4 above)
Comment (espenove) [reply app-directory/README.md]: removed in latest commit

File: context/src/examples/Contact.ts (hunk 6 above)
Comment (espenove) [reply context/src/examples/Contact.ts]: moved to example folder

**Review by rikoe — APPROVED** [review]

**Pull request merged**